### ISSUE TITLE
error when try to install php71-v8 on osx

### ISSUE THREAD

**Issue by aight8** [pinepain/homebrew-devtools#1]
it happens on **macOS sierra**

```
$ brew install php71-v8
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/php).
==> Updated Formulae
homebrew/php/codeception

==> Installing php71-v8 from pinepain/devtools
==> Installing dependencies for pinepain/devtools/php71-v8: v8@5.9
==> Installing pinepain/devtools/php71-v8 dependency: v8@5.9
==> Downloading https://github.com/v8/v8/archive/5.9.5.tar.gz
Already downloaded: /Users/sebi/Library/Caches/Homebrew/v8@5.9-5.9.5.tar.gz
==> Cloning https://chromium.googlesource.com/chromium/tools/depot_tools.git
Updating /Users/sebi/Library/Caches/Homebrew/v8@5.9--depot_tools--git
==> Checking out revision d7f5675931137d03f20e1b976bb672d23e109cf0
==> gclient root
==> gclient config --spec solutions = [
  {
    "url": "https://chromium.googlesource.com/v8/v8.git",
    "managed": False,
    "name": "v8",
    "deps_file": "DEPS",
    "custom_deps": {},
  },
]
target_os = [ 
==> gclient sync -vvv -j 8 -r 5.9.5
==> gn args out.gn/x64.release --list
Last 15 lines from /Users/sebi/Library/Logs/Homebrew/v8@5.9/04.x64.release:
2017-03-25 10:26:37 +0100

gn args out.gn/x64.release --list

ERROR Not a build directory.
This command requires an existing build directory. I interpreted your input
"out.gn/x64.release" as:
  /Users/sebi/Library/Caches/Homebrew/v8@5.9--v8--gclient/v8/out.gn/x64.release/
which doesn't seem to contain a previously-generated build.

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/pinepain/homebrew-devtools/issues
```

and ```brew install v8@5.9``` output something similar:

```
$ brew install v8@5.9
Updating Homebrew...
==> Installing v8@5.9 from pinepain/devtools
==> Downloading https://github.com/v8/v8/archive/5.9.5.tar.gz
Already downloaded: /Users/sebi/Library/Caches/Homebrew/v8@5.9-5.9.5.tar.gz
==> Cloning https://chromium.googlesource.com/chromium/tools/depot_tools.git
Updating /Users/sebi/Library/Caches/Homebrew/v8@5.9--depot_tools--git
==> Checking out revision d7f5675931137d03f20e1b976bb672d23e109cf0
==> gclient root
==> gclient config --spec solutions = [
  {
    "url": "https://chromium.googlesource.com/v8/v8.git",
    "managed": False,
    "name": "v8",
    "deps_file": "DEPS",
    "custom_deps": {},
  },
]
target_os = [ 
==> gclient sync -vvv -j 8 -r 5.9.5
==> gn args out.gn/x64.release --list
Last 15 lines from /Users/sebi/Library/Logs/Homebrew/v8@5.9/04.x64.release:
2017-03-25 10:28:40 +0100

gn args out.gn/x64.release --list

ERROR Not a build directory.
This command requires an existing build directory. I interpreted your input
"out.gn/x64.release" as:
  /Users/sebi/Library/Caches/Homebrew/v8@5.9--v8--gclient/v8/out.gn/x64.release/
which doesn't seem to contain a previously-generated build.

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/pinepain/homebrew-devtools/issues
```

thank you for helping!

infos:
```ls -l /Users/sebi/Library/Caches/Homebrew/v8\@5.9--v8--gclient/v8/out.gn/x64.release/```
the directory exists but is empty...

**Comment by pinepain** [comment]
Hi Sebastian! Thank you for reporting, it's clearly a bug. The fix is on its way, will test locally with clean cache and publish a fix.

**Comment by pinepain** [comment]
Fixed by 78d7ceb6bb1d09310a032308715c1e63b5071ea9
